woocommerce/woocommerce-android
# UI to display a list of onboarding tasks inside my Store tab

Closes: #8260 


### Description


UI to display a list of onboarding tasks inside my Store tab. Plain UI that doesn't support any interactions for now. The feature is under a feature flag that is only enabled for `debug` builds.

I've opened a discussion in design P2 pecCkj-kA-p2 about the following: 
- Where should we place the new onboarding tasks list: nested inside the tabs or outside the tabs. 
- Overflow button at the top right corner to display a single option, doesn't seem to make much sense. 

Until those 2 points are clarified I've added the list nested inside the tabs in My Store screen and also have not added the overflow menu.

### Testing instructions
Just launch the app and check that the UI matches design (with the expected differences between iOS and Android platforms): VyLr7LvKodHE4kINfBE7Lw-fi-1264%3A69768

![Screenshot 2023-01-30 at 12 56 04](https://user-images.githubusercontent.com/2663464/215470454-1a466db7-7fb7-4541-8d52-54c3c3950a00.png)
